client/README.md

```diff
@@ -1,31 +1,69 @@
-# Client
+# ODS Provisioning App Frontend Client
```

Comment:
I guess it should be either "ODS Provisioning App Frontend" or "ODS Provisioning App Client".